Wikipedia/Code/WMFExploreViewController.m

```diff
@@ -1581,9 +1582,39 @@ - (nullable UIViewController *)previewingContext:(id<UIViewControllerPreviewing>
         ((WMFArticleViewController *)vc).articlePreviewingActionsDelegate = self;
     }
 
+    [previewingContext.previewingGestureRecognizerForFailureRelationship addObserver:self
+                                                                          forKeyPath:kvo_WMFExploreViewController_peek_state_keypath
+                                                                             options:NSKeyValueObservingOptionNew
+                                                                             context:&kvo_WMFExploreViewController_peek_gesture_recognizer_for_failure_relationship];
+
     return vc;
 }
 
+static const NSString *kvo_WMFExploreViewController_peek_gesture_recognizer_for_failure_relationship = nil;
```

Comment:
@amrox pointed out that for debugging it's helpful to assign this to `@"kvo_WMFExploreViewController_peek_gesture_recognizer_for_failure_relationship"` - it's not that way other places in the codebase but would be good to do going forward (and fix as we encounter them)